tfupdate/option.go

```diff
@@ -48,13 +51,18 @@ func NewOption(updateType string, name string, version string, platforms []strin
 		regexps = append(regexps, r)
 	}
 
+	if sourceMatchType == "" {
+		sourceMatchType = "full"
+	}
```

Comment:
As mentioned above, could you check for regex compile errors here and store the compiled result in the option instance if there are no errors?

Reply:
This is fixed with the commit (https://github.com/minamijoyo/tfupdate/pull/108/commits/e1441c9a5653fb2a577701c7e6000539839dd874).